bedrock/settings/base.py

```diff
@@ -1326,6 +1336,22 @@ def JINJA_CONFIG():
         ),
         'output_filename': 'js/firefox_os-bundle.js',
     },
+    'firefox_os_new': {
+        'source_filenames': (
+            'js/base/mozilla-modal.js',
+            'js/libs/jquery.waypoints.min.js',
+            'js/libs/jquery.waypoints-sticky.min.js',
+            'js/libs/tweenmax.1.9.7.min.js',
```

Comment:
I though so to but, turns out the combination of the logic for the phone/signup and possible the inclusion of the get_device fragment requires those stay. Not sure about jquery.color though, that one might be able to be removed.